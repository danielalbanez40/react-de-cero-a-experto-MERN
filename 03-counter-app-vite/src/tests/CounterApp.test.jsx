import CounterApp from '../CounterApp';
import { render, screen } from '@testing-library/react';

describe('Tests in CounterApp', () => {
  const initialValue = 100;

  test('should make match with snapshot', () => {
    const { container } = render(<CounterApp value={initialValue} />);
    expect(container).toMatchSnapshot();
  });

  test('should show initial value from 100', () => {
    render(<CounterApp value={initialValue} />);
    expect(screen.getByText(100)).toBeTruthy();
  });
});
