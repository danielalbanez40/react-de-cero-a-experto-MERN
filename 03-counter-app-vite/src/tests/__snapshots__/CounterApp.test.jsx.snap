// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in CounterApp should make match with snapshot 1`] = `
<div>
  <h1>
    CounterApp
  </h1>
  <h2>
     
    100
     
  </h2>
  <button>
     +1 
  </button>
  <button>
     -1 
  </button>
  <button>
     Reset 
  </button>
</div>
`;
